manifests/02-deployment-ibm-cloud-managed.yaml

```diff
@@ -35,6 +35,10 @@ spec:
         - $(CANARY_IMAGE)
         - --release-version
         - $(RELEASE_VERSION)
+        - --gateway-api-operator-channel
+        - $(GATEWAY_API_OPERATOR_CHANNEL)
+        - --gateway-api-operator-version
+        - $(GATEWAY_API_OPERATOR_VERSION)
```

Comment:
To keep consistency with 02-deployment.yaml, it might be better quote these, and fix the other $(VALUE) values to be quoted as well, "$(VALUE)".

Reply:
I initially did that, but it failed the verify job. Running the `hack/update-profile-manifests.sh` script generates an unquoted version.